### PR TITLE
fix(readme): rewrite relative urls with github cdn paths

### PR DESCRIPTION
**Pull request by cpreston321**
Resolves: #24

### CODE REVIEW

File: server/routes/repos/[owner]/[repo]/readme.ts [hunk 1]
```diff
@@ -2,11 +2,33 @@ import { ghMarkdown } from "~/utils/github";
...
+    const repo = `${prefix}/${event.context.params.owner}/${event.context.params.repo}`;
+    let markdown = await $fetch(`${repo}/main/README.md`);
+    let html = await ghMarkdown(markdown, repo, "readme");
+
+    const pathReplacer = (
+      match: string,
+      link: string,
+      url: string | undefined,
+      url2: string
+    ) => {
```
Comment (pi0) [inline server/routes/repos/[owner]/[repo]/readme.ts:15]: Perf: We could define a normal function outside for this.

Comment (cpreston321) [reply server/routes/repos/[owner]/[repo]/readme.ts]: @pi0  I defined a helper function outside that can be used elsewhere. Let me know if you have any more suggestions! 

Thanks,
CP

**Comment by pi0** [comment]
Thanks and sorry took so long to check. I have rebased your PR + simplified the implementation.

File: utils/markdown.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  /(?<link>\[.*?]\((?<url>.*?)\)|<img.*?src="(?<url2>.*?)".*?>)/g;
+
+/**
+ * HTML & Mardown Function to replace relative image paths with absolute paths
+ */
+export function resolveMarkdownRelativeLinks(
+  content: string,
+  cdnBaseURL: string,
+) {
+  console.log("fixing", content);
```
Comment (cpreston321) [inline utils/markdown.ts:11]: @pi0 Should we remove this comment?

**Pull request merged**